apps/worker/services/report/languages/tests/unit/test_lcov.py

```diff
@@ -208,3 +208,114 @@ def test_regression_partial_branch(self):
                 (1047, "1/2", "b", [[0, "1/2", ["0:0"], None, None]], None, None),
             ]
         }
+
+    def test_lcov_partials_as_hits_enabled(self):
+        """Test that partial branches become hits when partials_as_hits=True"""
+        lcov_data = b"""
+TN:
+SF:partial_test.c
+DA:10,5
+BRDA:10,0,0,1
+BRDA:10,0,1,0
+end_of_record
+"""
+
+        def path_fixer(path):
```

Comment:
I see this is repeated in all tests, should we move this to the top and have it only written once?